cyctl/go.mod

```diff
@@ -1,74 +1,127 @@
 module github.com/cyclops-ui/cycops-cyctl
 
-go 1.22.0
+go 1.24.0
```

Comment:
Can you revert to the previous version of Go and other dependencies unless crucial?